develop/index.md

```diff
@@ -12,7 +12,9 @@ If you would like to help, here are a few things you can do:
 
 - **Core development**: Can you speak C++? The [XCSoar Developer Manual]({{ site.download_server_url }}/{{ site.xcsoar_stable_version }}/XCSoar-developer-manual.pdf) has instructions on how to get the source code, hack it, and submit your patches.  You can contact other developers [here](/contact/).
 You can [pick an issue](https://github.com/XCSoar/XCSoar/issues) and make a pull request.
-- **Testing**: [Download](https://xcsoar.org/download/) XCSoar, test it, and [report bugs](/develop/new_ticket.html).
+- **Testing**: 
+	- [Download the released version](https://xcsoar.org/download/) of XCSoar, test it, and [report bugs](/develop/new_ticket.html).
+	- [Download the nightly-build version](https://download.xcsoar.org/nightly_builds/) of XCSoar, test it, and [report bugs](/develop/new_ticket.html). You will be at the cutting edge of XCSoar development. This version may not be stable, but you will be able to try the new features and eventually report recently-introduced bugs.
```

Comment:
Great. :) Please rephrase this so that its _very_ clear that these are unstable and should not be used for flight.

Reply:
Thank you for your comment.
It is now underlined that the nightly builds are not for flight.